lms/resources/_js_config/__init__.py

```diff
@@ -55,6 +55,7 @@ def add_document_url(self, document_url):
         if document_url.startswith("blackboard://"):
             self._config["api"]["viaUrl"] = {
                 "authUrl": self._request.route_url("blackboard_api.oauth.authorize"),
+                "product": self._request.product.value,
```

Comment:
Send the product to the frontend so we'll receive it back and we avoid the need to guess the product without the context of an LTI launch.